docs/user/hbase/configuration.rst

```diff
@@ -51,6 +69,18 @@ Disable remote filtering. Remote filtering and coprocessors speed up queries, ho
 of custom JARs in HBase. Since this is not always possible, they can be disabled by setting this to ``false``.
 This property will be overridden by the data store configuration parameter, if both are specified.
 
+geomesa.hbase.scan.buffer
++++++++++++++++++++++++++
+
+Specify the maximum number of results to pre-buffer in local memory when executing a scan, if the client is not
+consuming the results as fast as they are being returned.
+
+geomesa.hbase.table.availability.timeout
```

Comment:
Were these two configs just missing in the docs?  If so, thanks for adding them!

Reply:
yeah i went through and made sure we had an entry for all the sys props.